src/Common/tests/System/Drawing/Helpers.cs

```diff
@@ -20,31 +20,7 @@ public static class Helpers
         public const string AnyInstalledPrinters = nameof(Helpers) + "." + nameof(IsAnyInstalledPrinters);
         public const string WindowsRS3OrEarlier = nameof(Helpers) + "." + nameof(IsWindowsRS3OrEarlier);
 
-        public static bool GetGdiplusIsAvailable()
-        {
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                return PlatformDetection.IsNotWindowsNanoServer && PlatformDetection.IsNotWindowsServerCore;
-            }
-            else
-            {
-                IntPtr nativeLib;
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-                {
-                    nativeLib = dlopen("libgdiplus.dylib", RTLD_NOW);
-                }
-                else
-                {
-                    nativeLib = dlopen("libgdiplus.so", RTLD_NOW);
-                    if (nativeLib == IntPtr.Zero)
-                    {
-                        nativeLib = dlopen("libgdiplus.so.0", RTLD_NOW);
-                    }
-                }
-
-                return nativeLib != IntPtr.Zero;
-            }
-        }
+        public static bool GetGdiplusIsAvailable() => PlatformDetection.IsDrawingSupported;
```

Comment:
Why do we need this at all now?  Can't call sites just use PlatformDetection.IsDrawingSupported?

Reply:
No. Because call sites use multi conditions, some from Helpers and some from PlatformDetection. We currently don't support multi type conditions in ConditionalFact and ConditionalTheory.

Reply:
Then at least shouldn't this be called IsDrawingSupported?

Reply:
Yes I agree we should rename this to IsDrawingSupported.